server/tests/api_shared_test.py

```diff
@@ -47,3 +47,22 @@ def test_cached_name(self, mock_data_fetcher):
 
         result = shared_api.cached_name('^'.join([dcid1, dcid2, dcid3]))
         assert result == {dcid1: 'California', dcid2: '', dcid3: 'Colorado'}
+
+
+class TestIsFloat(unittest.TestCase):
+
+    def test_is_float(self):
+        cases = [{
+            'query': 'abc',
+            'expected': False
+        }, {
+            'query': '1.26',
+            'expected': True
+        }, {
+            'query': '-',
+            'expected': False
+        }]
```

Comment:
Also add a test for 0.0 and a non-float, but numeric value?

Reply:
done